cadccutout/cadccutout/core.py

```diff
@@ -148,8 +150,8 @@ def __init__(self, helper_factory=FileHelperFactory(),
         self.helper_factory = helper_factory
         self.input_range_parser = input_range_parser
 
-    def cutout(self, cutout_dimensions, input_reader=sys.stdin,
-               output_writer=sys.stdout, file_type='FITS'):
+    def cutout(self, cutout_dimensions, input_reader=UNSET_ARG,
```

Comment:
I would make the defaults None. You avoid doing this just to tell the caller he's wrong if he sets them to None. I don't think it's worth.